content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -5463,3 +5472,84 @@ const createUserAndPost = (
   })
 }
 ```
+
+## Compare columns in the same table
+
+From version 4.3.0, you can compare columns in the same table directly, for non-unique filters.
+
+<Admonition type="info">
+
+In the following situations, you must [use raw queries to compare columns in the same table](/guides/database/troubleshooting-orm/help-articles/comparing-columns-through-raw-queries):
+
+- If you use a version earlier than 4.3.0.
+- If you want to use a unique filter, such as `findUnique` or `findUniqueOrThrow`. **(TBD: does this apply to fields with @@unique set on them?)**
```

Comment:
Yes, unique filters in general don't support field ref

Reply:
Thanks!